core/src/main/scala/Jwt.scala

```diff
@@ -59,6 +60,7 @@ object Jwt extends JwtCore[String, String] {
   *
   */
 trait JwtCore[H, C] {
+  implicit protected val clock: Clock = Clock.systemUTC
```

Comment:
I don't remember much about implicit, but can you actually override the clock even if you define an implicit here, inside the main trait?

Reply:
Depend on what you mean by override it.
Impls of this trait can override it, as you can see in all of the new classes with `override val clock` I've added. You can't override in the objects after they've been initialized, since they're singletons.